client/client-multi/client-c/toolchains/esp8266.cmake

```diff
@@ -20,46 +20,46 @@
 
 include(CMakeForceCompiler)
 
-set(ARM_GCC_COMPILER "xtensa-lx106-elf-gcc${CMAKE_EXECUTABLE_SUFFIX}")
+set(XTENSA_GCC_COMPILER "xtensa-lx106-elf-gcc${CMAKE_EXECUTABLE_SUFFIX}")
 
-if(DEFINED ENV{ESP8266_TOOLCHAIN_PATH})
-  set(TOOLCHAIN_PATH "$ENV{ESP8266_TOOLCHAIN_PATH}")
-  message(STATUS "Toolchain path is provided: ${TOOLCHAIN_PATH}")
-else ()
+if (NOT DEFINED ESP8266_TOOLCHAIN_PATH)
   # Check if GCC is reachable.
-  find_path(TOOLCHAIN_PATH bin/${ARM_GCC_COMPILER})
+  find_path(ESP8266_TOOLCHAIN_PATH bin/${XTENSA_GCC_COMPILER})
 
-  if (NOT TOOLCHAIN_PATH)
+  if (NOT ESP8266_TOOLCHAIN_PATH)
```

Comment:
Let's change a style - remove spaces between command name and parenthesis, e.g.:
`if (` => `if(`